prow/cmd/config-bootstrapper/main.go

```diff
@@ -55,6 +57,8 @@ func gatherOptions() options {
 	fs.StringVar(&o.jobConfigPath, "job-config-path", "", "Path to prow job configs.")
 	fs.StringVar(&o.pluginConfig, "plugin-config", "/etc/plugins/plugins.yaml", "Path to plugin config file.")
 
+	fs.BoolVar(&o.gzip, "gzip", false, "Wether or not to GZIP the configmap keys")
```

Comment:
Err please let's make this per-entry not global. We definitely use `config-updater` for way more than Prow config

Reply:
The bootstrapper only supports Prow configs (?)

Reply:
I see config.yaml, job-configs, and plugins.yaml

Reply:
(nit: whether)

Reply:
https://github.com/kubernetes/test-infra/pull/11756